ui/pages/confirmations/components/approve-content-card/approve-content-card.js

```diff
@@ -154,9 +157,7 @@ export default function ApproveContentCard({
                 color={TextColor.textAlternative}
                 as="h6"
               >
-                {isSetApproveForAll
-                  ? t('functionSetApprovalForAll')
-                  : t('functionApprove')}
+                {`Function: ${functionName}`}
```

Comment:
Do we need translactions here? Maybe not for the fucntion name, but for the word `Function`?